src/rebar3_format_prv.erl

```diff
@@ -123,8 +122,8 @@ format(Files, Formatter, Opts) ->
                 {error, {unformatted_files, ChangedFiles}}
           end
     catch
-      _:{cant_parse, File, {_, erl_parse, Error}} ->
-          rebar_api:debug("Couldn't parse ~s: ~p", [File, Error]),
+      _:{cant_parse, File, {Line, erl_parse, Error}} ->
+          rebar_api:debug("Couldn't parse ~s:~B ~p", [Line, File, Error]),
```

Comment:
Why not `~p`, just in case? And we can totally use `warn` here.
```suggestion
          rebar_api:warn("Couldn't parse ~s:~p ~p", [Line, File, Error]),
```

Reply:
Sure, I don't mind.